Source/Fuse.Navigation/RouterModify.uno

```diff
@@ -98,18 +98,10 @@ namespace Fuse.Navigation
 		internal enum Flags
```

Comment:
Is this in use still? Only one member and internal?

Reply:
I had removed it at first and replaced it with a `bool`, but the flags looked better, so I replaced it. We still need the two constructors, just so we can emit a deprecation warning on the one.